[2023-11-11T14:42:02.693+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ProjetETL.extraction_fichier2 manual__2023-11-11T14:41:52.863981+00:00 [queued]>
[2023-11-11T14:42:02.715+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ProjetETL.extraction_fichier2 manual__2023-11-11T14:41:52.863981+00:00 [queued]>
[2023-11-11T14:42:02.717+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-11T14:42:02.746+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extraction_fichier2> on 2023-11-11 14:41:52.863981+00:00
[2023-11-11T14:42:02.755+0000] {standard_task_runner.py:57} INFO - Started process 23303 to run task
[2023-11-11T14:42:02.762+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ProjetETL', 'extraction_fichier2', 'manual__2023-11-11T14:41:52.863981+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/Projet.py', '--cfg-path', '/tmp/tmpl1wq1ntx']
[2023-11-11T14:42:02.766+0000] {standard_task_runner.py:85} INFO - Job 53: Subtask extraction_fichier2
[2023-11-11T14:42:02.867+0000] {task_command.py:416} INFO - Running <TaskInstance: ProjetETL.extraction_fichier2 manual__2023-11-11T14:41:52.863981+00:00 [running]> on host fb26c01043df
[2023-11-11T14:42:03.035+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ProjetETL' AIRFLOW_CTX_TASK_ID='extraction_fichier2' AIRFLOW_CTX_EXECUTION_DATE='2023-11-11T14:41:52.863981+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-11T14:41:52.863981+00:00'
[2023-11-11T14:42:03.044+0000] {python.py:194} INFO - Done. Returned value was: Erreur lors de l'extraction du fichier : [Errno 2] No such file or directory: 'Users/celinahamzaoui/Desktop/Cours_Sup_de_vinci/Datawarehouse/TP3/TP_Airflow/data/departements-region.json'
[2023-11-11T14:42:03.089+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ProjetETL, task_id=extraction_fichier2, execution_date=20231111T144152, start_date=20231111T144202, end_date=20231111T144203
[2023-11-11T14:42:03.137+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-11T14:42:03.190+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
